Application form

SimonWoodhouse73 opened the pull request.

Emjem125 committed:
Header changed, Profile duplicated

Decided that the Nav Bar looks better without Logo.
Tried to sort out the header text.
Added in more of the profile changing the text but haven't managed to put them on the same line.

SimonWoodhouse73 committed:
Added skeleton application form

basic application form added for you to style up.
Fixed header text moving issue.

Needs code in the form to respond to Apply button click

Emjem125 reviewed (approved):
The form is good, do you know how to make it say like form submitted when it is completed?

Emjem125 committed:
Added image/text/button

Added in all necessary info, images and buttons.
Buttons not aligned next to each other.
Changed Drop Down Menu info

Emjem125 committed:
Removed Extra on design

Emjem125 committed:
Added wording
